hatchet_sdk/context/context.py

```diff
@@ -218,33 +228,34 @@ def __init__(
         else:
             self.input = self.data.get("input", {})
 
-    def step_output(self, step: str):
+    def step_output(self, step: str) -> dict[str, Any]:
         try:
-            return self.data["parents"][step]
+            return cast(dict[str, Any], self.data["parents"][step])
         except KeyError:
             raise ValueError(f"Step output for '{step}' not found")
 
     def triggered_by_event(self) -> bool:
-        return self.data.get("triggered_by", "") == "event"
+        return cast(str, self.data.get("triggered_by", "")) == "event"
 
-    def workflow_input(self):
+    def workflow_input(self) -> dict[str, Any]:
         return self.input
 
-    def workflow_run_id(self):
+    def workflow_run_id(self) -> str:
         return self.action.workflow_run_id
 
-    def cancel(self):
+    def cancel(self) -> None:
         logger.debug("cancelling step...")
         self.exit_flag = True
 
     # done returns true if the context has been cancelled
-    def done(self):
+    def done(self) -> bool:
         return self.exit_flag
 
-    def playground(self, name: str, default: str = None):
+    def playground(self, name: str, default: str | None = None) -> str | None:
         # if the key exists in the overrides_data field, return the value
         if name in self.overrides_data:
-            return self.overrides_data[name]
+            ## TODO: Check if this is the right type
+            return cast(str, self.overrides_data[name])
```

Comment:
checking on this! is`overrides_data[name]` a string?

Reply:
I'm honestly not sure -- but this is deprecated and undocumented anyway, so perhaps just add a deprecation warning and we can cast to a string.